basic/exercise1/script.js

```diff
@@ -9,10 +10,12 @@ function formatDate(date) {
     var hours = date.getHours();
     var ampm = hours >= 12 ? 'PM' : 'AM';
     hours = hours % 12;
+    var formatHours = formatAsTwoDigits(hours);
     var minutes = formatAsTwoDigits(date.getMinutes());
     var seconds = formatAsTwoDigits(date.getSeconds());
-    var time = `${weekDays} ${formatAsTwoDigits(hours)} ${ampm} ${minutes}: ${seconds}`;
+    var time = `${weekDays} ${formatHours} ${ampm} ${minutes}: ${seconds}`;
     return time;
+
```

Comment:
Why did you add empty line here?
It is less readable
I meant at the end of the function - which means after closing bracket.